build/tradable.js

```diff
@@ -0,0 +1,26 @@
+const builtUntradable = ['Warframe', 'Throwing', 'Shotgun', 'Rifle', 'Pistol', 'Melee', 'Sword And Shield']
+const tradableConditions = (i) => !(builtUntradable.includes(i.type) && i.name.match(/Prime/ig))
```

Comment:
Could you rename i to item? as it's commonly associated to index.
```suggestion
const tradableConditions = (item) => !(builtUntradable.includes(item.type) && item.name.match(/Prime/ig))
```

Reply:
sure. i'll do it manually since i'm doing a few changes alongside.